### PR TITLE
Added docs for execute action - result status

### PR DESCRIPTION
**Pull request by lsheikal**
Added docs for execute action - result status
    - To change the result status when the step is passed

### CODE REVIEW

File: docs/blitz/design/actions/actions_list.rst [hunk 1]
```diff
@@ -42,6 +42,41 @@ Example can be seen below.
         device: PE1
         timeout: 10
 
+The user can now change the result status to failed, passed, aborted, blocked, skipped, errored
+and passx, when the step is passed. In the below example we assigned result_status to passx.
+when only this step is passed, the result will be changed to ``passx``.
```
Comment (domachad) [inline docs/blitz/design/actions/actions_list.rst:47]:
```suggestion
The user can now change the result status to failed, passed, aborted, blocked, skipped, errored
or passx when the step is passed. In the example below, result_status is changed to passx.
```

File: docs/blitz/design/actions/actions_list.rst [hunk 2]
```diff
@@ -42,6 +42,41 @@ Example can be seen below.
...
+    - execute: # ACTION
+        # (Either device hostname or device alias)
+        device: R1
+        # Send show version to the device
+        command: show version
+        # To Change the result status
+        result_status: passx
+
+In the below example when either include or exclude keyword is passed its result will be changed
+to ``aborted``.
```
Comment (domachad) [inline docs/blitz/design/actions/actions_list.rst:60]:
```suggestion
In the example below, if include or exclude conditions are met, their results will be changed to ``aborted``. 
```

**Review by domachad — CHANGES_REQUESTED** [review]
I've made some suggestions. I think you can elaborate the paragraphs a bit better.

**Review by domachad — APPROVED** [review]

File: docs/blitz/design/actions/actions_list.rst (hunk 1 above)
Comment (lsheikal) [reply docs/blitz/design/actions/actions_list.rst]: updated!!!

File: docs/blitz/design/actions/actions_list.rst (hunk 2 above)
Comment (lsheikal) [reply docs/blitz/design/actions/actions_list.rst]: updated!!!

Comment (tahigash) [reply docs/blitz/design/actions/actions_list.rst]: please change from `aborted` to `failed`. usual case would be changing to `passx` or `failed`. `aborted` or `errored`, user can specify, but not usual...

**Review by tahigash — COMMENTED** [review]
left comments

File: docs/blitz/design/actions/actions_list.rst (hunk 2 above)
Comment (lsheikal) [reply docs/blitz/design/actions/actions_list.rst]: updated

**Review by tahigash — APPROVED** [review]

**Pull request merged**